security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -20,6 +20,7 @@ Issued: 2023-10-02
 The following advisory is available for the {cert-manager-operator} 1.12.0:
 
 * link:https://access.redhat.com/errata/RHEA-2023:5339[RHEA-2023:5339]
+* link:https://access.redhat.com/errata/RHBA-2023:5412[RHBA-2023:5412]
```

Comment:
If there are now two advisories associated with version 1.12.0, I think link 20 needs to be updated advisory/advisories.